sycl/test/sub_group/load_store.cpp

```diff
@@ -180,6 +172,7 @@ int main() {
     check<aligned_short, 4>(Queue);
     check<aligned_short, 8>(Queue);
     if (Queue.get_device().has_extension("cl_khr_fp16")) {
+      typedef half aligned_half __attribute__((aligned(16)));
```

Comment:
What about `using` instead in 2019?

Reply:
This test uses `typedef` in all places so I used it to save one style. I think changing all `typedefs` to `usings` not in scope of this PR :)